REQUIRE

```diff
@@ -1,2 +1,2 @@
-julia 0.4
+julia 0.7
 Compat 0.17
```

Comment:
are their any remaining calls to @compat ?  if not can we remove this requirement?